Skype/SDN/articles/burstgapdensity-element-qualitypropertiestype-complextype-skype-sdn-2-2-c.md

```diff
@@ -1,16 +1,17 @@
 ﻿---
-title: BurstGapDensity element (QualityPropertiesType complexType) 
+title: BurstGapDensity element (QualityPropertiesType complexType) Schema C
```

Comment:
Let's put a parenthesis around Schema C in the title line as well, this will give the article's title and H1 title a unifying look. (Schema C)